### ISSUE TITLE
500 Server Error when importing egg

### ISSUE THREAD

**Issue by ManucrackYT** [Fex-Development/Fex-multiegg#2]

**Comment by purejosh** [comment]
@Nightmaregodss Any updates? Is this abandoned? If so, please archive the github project.

**Comment by ManucrackYT** [comment]
> @Nightmaregodss Any updates? Is this abandoned? If so, please archive the github project. 
> 

Yes it is, I asked once to the dev and one year later there is no modification in the repo
